docs/networkpolicy-recommendation.md

```diff
@@ -0,0 +1,148 @@
+as Theia's NetworkPolicy Recommendation feature needs to process pod label information.
+
+Then, please follow these [deployment steps](network-flow-visibility.md#deployment-steps)
+to deploy Theia.
+
+Commet out following lines since user won't need to deploy Spark Operator separately after PR#31 merged.
+
+<!-- After we finish the deployment of Grafana Flow Collector and can see some flow records in the Grafana UI,
+we can continue to deploy other necessary components of policy recommendation feature:
+
+We need to install the [Kubernetes Operator for Apache Spark](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator)
+in the cluster. Our policy recommendation logic is implemented as a [Spark](https://github.com/apache/spark)
+application, the Kubernetes Operator for Apache Spark helps us schedule the Spark job in the Kubernetes cluster.
+
+Please run the following commands:
+
+```bash
+helm repo add spark-operator https://googlecloudplatform.github.io/spark-on-k8s-operator
+helm install policy-reco spark-operator/spark-operator --namespace flow-visibility --set image.tag=latest
+```
+
+This will install the Kubernetes Operator for Apache Spark into the `flow-visibility` Namespace.
+
+Once we finish these deployment steps, we should see the `policy-reco-spark-operator` Pod running inside the `flow-visibility` namespace:
+
+```bash
+$ kubectl get pods -n flow-visibility
+NAME                                          READY   STATUS    RESTARTS   AGE
+policy-reco-spark-operator-56c4cb454c-4vhfh   1/1     Running   0          2m19s
+``` -->
+
+## Perform NetworkPolicy Recommendation
+
+Currently, the NetworkPolicy recommendation feature only supports command line interaction through `theia`.
+`theia` is the command-line tool which provides access to Theia network flow visibility capabilities.
+To get more information about `theia`, please refer to this [doc](theia.md).
+
+We have 3 `theia` commands for the policy recommendation feature:
+
+- `theia policy-recommendation run`
+- `theia policy-recommendation status`
+- `theia policy-recommendation retrieve`
+
+Or you could use `pr` as a short alias of `policy-recommendation`:
+
+- `theia pr run`
+- `theia pr status`
+- `theia pr retrieve`
+
+To see all options and usage examples of these commands, you may run `theia policy-recommendation [subcommand] --help`.
+
+### Run a policy recommendation job
+
+The `theia policy-recommendation run` command can run a new policy recommendation job.
+If the new policy recommendation job is created successfully, the `recommendation ID` of this job will be returned:
+
+```bash
+$ theia policy-recommendation run
+Successfully created policy recommendation job with ID e998433e-accb-4888-9fc8-06563f073e86
+```
+
+A policy recommendation job may take a few minutes to an hour to complete
```

Comment:
We may need to state this command by default won't wait until the job is finished. If customers want to wait, they can add an option?